File: Views/src/components/out-sourced/collect-deliver/dto/destiny-dto.ts
import { CustomerDto } from "../../../../components/main/customer/components/commons-components/dtos/customer-dto";
import { PartnerDto } from "src/components/main/partner/dtos/partner-dto";
import { RootBase } from "../../../../shared/entities-dtos/root-base";

export class DestinyDto extends RootBase {

  customerId: number | null;
  customer: CustomerDto | null;
  partnerId: number | null;
  partner: PartnerDto | null;
  noRegisterName: string | null;
  noRegisterAddress: string | null;
  description: string;
  collectDeliverId: number;
  //collectDeliver: CollectDeliverDto;
}

File: Application/Services/Operations/Outsourced/Dtos/DestinyDto.cs

using Application.Services.Operations.Main.Customers.Dtos;
using Application.Services.Operations.Main.Partners.Dtos;
using Application.Services.Shared.Dtos;

namespace Application.Services.Operations.Outsourced.Dtos
{
    public class DestinyDto : RootBaseDto
    {
       public int? CustomerId { get; set; }
        public CustomerDto Customer { get; set; }
        public int? PartnerId { get; set; }
        public PartnerDto Partner { get; set; }
        public string NoRegisterName { get; set; }
        public string NoRegisterAddress { get; set; }

    }
}
